designs/developer/add-45/sections/pipelines/pipelines.md

```diff
@@ -0,0 +1,33 @@
+---
+parent: Developer
+version: 4.5
+---
+
+# Pipelines section in Add Flows- Updates
```

Comment:
Should this still say "updates"? Or should it remain it's normal title since it will ultimately be the design for pipelines within the add section?

Reply:
I was leaving the 'updates' since just a couple things have been changed since the 4.3 pipelines. I can get rid of it if it doesn't make sense to others.

Reply:
I think it makes sense to leave it out and assume all documentation is the latest and greatest "Update". But it totally depends on the strategy that's being taken on the Developer side of the docs. Probably best to check with @serenamarie125 @beaumorley for their guidance.

Reply:
I think you want to merge the documentation from the 4.3 release with what you have here. So you would take the content from the 4.3 page (openshift-origin-design/designs/developer/add-43/sections/pipelines/pipelines.md) that is still relevant and add the new content removing the stuff from the 4.3 doc that has changed. The 4.3 documentation will remain under 4.3 and the new version with your updates will be in the 4.5 directory. No need to call it and update.